File: help/eng_gb/exercicios_da_ficha_3/Ficha03_002/Ficha03_002.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Value of X    : " ;
    $x = floatval(isset($_POST["x"])? $_POST["x"] : isset($_GET["x"]) ? $_GET["x"] : 0);
    
    echo $x . "<br/>";// print input 
    echo  "\nRoot         = " . ( sqrt( $x )  ) ;
    echo  "\nPotency     = " . ( pow( $x , 0.5 )  ) ;
    echo  "\nOperator     = " . ( $x ^ 0.5  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:55

?>

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_002/Ficha03_002.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_002 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Value of X\t: " );
        double x = Double.valueOf(keyb.nextLine());
        System.out.print("\nRoot \t\t= " + Math.sqrt( x ) );
        System.out.print("\nPotency \t= " + Math.pow( x , 0.5 ) );
        System.out.print("\nOperator \t= " + Math.pow( x , 0.5 ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:55